README.md

```diff
@@ -33,6 +33,10 @@ Run the server using:
 npm start
 ```
 
+Configuration of the server can be edited in the `./config.yml`.
+
+Optionally prepend the command with `VROOM_ROUTER=<router>`, `router` being `osrm` (default) or `ors`. Using the environment variable **will override the config.yml `router` setting**.
```

Comment:
`libosrm` is still a valid option, we should probably mention it too.

Reply:
right, will do